src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -172,16 +180,10 @@ public boolean processQuery(String query, ImportInspector dialog, OutputPrinter
 
         } catch (MalformedURLException e) {
             LOGGER.warn("Bad URL", e);
-        } catch (ConnectException e) {
+        } catch (ConnectException | UnknownHostException e) {
             status.showMessage(Localization.lang("Connection to IEEEXplore failed"), DIALOG_TITLE,
```

Comment:
Could we introduce the typical `"Connection to %0 failed", "IEEE"`? - See https://github.com/JabRef/jabref/pull/948. Maybe in a separate PR, but I just wanted that this is not forgotten.